shell/platform/linux/fl_backing_store_provider.cc

```diff
@@ -82,7 +82,8 @@ uint32_t fl_backing_store_provider_get_gl_format(FlBackingStoreProvider* self) {
   // In Linux kN32_SkColorType is assumed to be kBGRA_8888_SkColorType.
   // So we must choose a valid gl format to be compatible with surface format
   // BGRA8.
-  // Following logics are copied from Skia GrGLCaps.cpp.
+  // Following logics are copied from Skia GrGLCaps.cpp:
```

Comment:
Entirely non-critical but may as well fix it while you're in here: "Following logic is copied"

Reply:
Fixed!